v23.1/transaction-retry-error-reference.md

```diff
@@ -97,13 +143,13 @@ The `RETRY_SERIALIZABLE` error occurs in the following cases:
 _Action_:
 
 1. If you encounter case 1 or 2 above, the solution is to:
-   1. Retry transaction _A_ as described in [client-side retry handling](transactions.html#client-side-intervention).
+   1. Retry transaction _A_ as described in [client-side retry handling](#client-side-retry-handling).
    1. Design your schema and queries to reduce contention. For more information about how contention occurs and how to avoid it, see [Transaction Contention](performance-best-practices-overview.html#transaction-contention). In particular, if you are able to [send all of the statements in your transaction in a single batch](transactions.html#batched-statements), CockroachDB can usually automatically retry the entire transaction for you.
    1. Use historical reads with [`SELECT ... AS OF SYSTEM TIME`](as-of-system-time.html).
 
 1. If you encounter case 3 above, the solution is to:
    1. Increase the `kv.closed_timestamp.target_duration` setting to a higher value. As described above, this will impact the freshness of data available via [Follower Reads](follower-reads.html) and [CDC changefeeds](change-data-capture-overview.html).
-   1. Retry transaction _A_ as described in [client-side retry handling](transactions.html#client-side-intervention).
+   1. Retry transaction _A_ as described in [client-side retry handling](#client-side-retry-handling).
    1. Design your schema and queries to reduce contention. For more information about how contention occurs and how to avoid it, see [Transaction Contention](performance-best-practices-overview.html#transaction-contention). In particular, if you are able to [send all of the statements in your transaction in a single batch](transactions.html#batched-statements), CockroachDB can usually automatically retry the entire transaction for you.
```

Comment:
Can't we just link to this entire section? https://deploy-preview-16280--cockroachdb-docs.netlify.app/docs/dev/transaction-retry-error-reference.html#minimize-transaction-retry-errors